### ISSUE TITLE
Allow group/assurances to be updated in Airtable when OFN listing is updated.

### ISSUE THREAD

**Issue by karlafalk** [openfoodfoundation/sib-discovery-components#83]
Thanks @mkllnk for this last[ n8n workflow ](https://community.openfoodnetwork.org/t/n8n-workflows-for-the-discover-regenerative-api/2954/10)that allows updates to flow from OFN to the Airtable/directory.  

I have now been able to get it working for the Waterloo project.  However, it looks like the ability to 'update' group membership/assurance has not been connected in the workflow, and was only added in concept.  So if an user removes themselves from a group on OFN, then after triggering the 'update listing' workflow, this change wouldn't correspondingly be reflected in Airtable.
 ![Image](https://github.com/user-attachments/assets/e76189a5-f03f-488b-b1cf-8f9049ea74eb)
 

Just wondering there are any updates on the completion of this step?  Are there plans to connect this part of the workflow for the AUS project?  It is not critical for the Waterloo Project, but we wanted to check the status of this element that would allow the listing to be fully updated in Airtable. Thanks!

**Comment by mkllnk** [comment]
Good question! We never completed that. I guess that those assurances are handled manually in Airtable. I'll ask the team.

**Comment by David-OFN-CA** [comment]
Cheers @mkllnk - just trying to close the loop on this so, did the Aus team have any feedback to offer on this point?

**Comment by mkllnk** [comment]
Yes, all handled manually for now. We need more experience with the partners to find a better process.
